doc/content/guides/getting-started/console/_index.md

```diff
@@ -43,26 +42,6 @@ Your application will be created and you will be redirected to the application o
 
 Devices are created within applications.
 
-### Link application
```

Comment:
I don't think we should remove this completely. We could split it into an **Create Application API Key** part and a **Link Application** part.

The **Create Application API Key** will be needed because people will still want to create API keys for other purposes.

The **Link Application** part will also still be needed because in a multi-region network, people may also want to link the AS of another region to the NS of that region.